src/adcm_pytest_plugin/docker_utils.py

```diff
@@ -234,6 +241,31 @@ def _init_adcm_cli(self):
         if not self._adcm_cli:
             self._adcm_cli = ADCMClient(url=self._adcm.url, **self.adcm_api_credentials)
 
+    def _generate_certs(self):
+        if not self.generate_certs:
+            return
+        self._certs_tmpdir = TemporaryDirectory()  # pylint: disable=consider-using-with
+        tmpdir = self._certs_tmpdir.name
+        os.system(
+            f"openssl req -x509 -newkey rsa:4096 -keyout {tmpdir}/key.pem -out {tmpdir}/cert.pem"
+            ' -days 365 -subj "/C=RU/ST=Moscow/L=Moscow/O=Arenadata Software LLC/OU=Release/CN=ADCM"'
+            f' -addext "subjectAltName=DNS:localhost,IP:127.0.0.1,IP:{self._adcm.ip}" -nodes'
+        )
+        file = io.BytesIO()
+        with tarfile.open(mode="w:gz", fileobj=file) as tar:
+            tar.add(tmpdir, "")
```

Comment:
Why do you specify the second argument?

Reply:
the default value is a filename:
```
archive.tar/
   tmpdir/
      key.pem
      cert.pem
```
empty string allows adding file in root archive directory
```
archive.tar/
   key.pem
   cert.pem
```